src/Cell.cpp

```diff
@@ -944,6 +1148,15 @@ std::string Cell::toString() {
   else
     string << ", type = UNFILLED";
 
+  if (_rotated) {
```

Comment:
What do you think about having parentheses around the rotation and translation elements in the string? Also, what about line breaks after the rotation and translation substrings to break them up? I feel like this would make it more readable.